eras/conway/test-suite/src/Test/Cardano/Ledger/Conway/Serialisation/Roundtrip.hs

```diff
@@ -23,5 +32,25 @@ allprops =
   testGroup
     (show $ typeRep (Proxy @e))
     [ testProperty "ConwayGenesis" $
-        roundTripCborExpectation @(ConwayGenesis (EraCrypto e)) (eraProtVerHigh @e)
+        roundTripCborExpectation @(ConwayGenesis (EraCrypto e)) (eraProtVerHigh @e),
+      testProperty "conway/TxOut with pointer addresses" ptrAddrTxOutNotDeserialized,
+      testProperty "conway/TxOut without pointer addresses" nonPtrAddrTxOutDeserialized
     ]
+
+ptrAddrTxOutNotDeserialized :: Property
+ptrAddrTxOutNotDeserialized =
+  forAll (genPtrAddress @(EraCrypto Conway)) $ \ptrAddr txOut ->
+    do
+      let ptrAddrTxOut = txOut & addrTxOutL .~ ptrAddr
+      let messageMatch s = "Pointer addresses not supported starting with version 9" `isInfixOf` s
```

Comment:
If we don't add these explanation we'll not know why exactly our test is failing
```suggestion
      let messageMatch s = counterexample ("Unexpected decoder failure message: " ++ s) $
                                              "Pointer addresses not supported starting with version 9" `isInfixOf` s
```